tests/execution/grpc-url-from-upstream.md

```diff
@@ -75,7 +75,7 @@ type News {
 ```yml
 - request:
     method: POST
-    url: http://localhost:50051/news.NewsService/GetAllNews
+    url: http://localhost:50051/NewsService/GetAllNews
```

Comment:
Here is the actual expected URL. 

<img width="679" alt="Screenshot 2024-02-16 at 7 26 06 PM" src="https://github.com/tailcallhq/tailcall/assets/194482/745a3319-06af-4736-8393-08734a8c9670">

Reply:
But where do we have `a.b.c.d.` on the config?